### PR TITLE
Resolved  https://github.com/webpack/css-loader/issues/102

### PR DESCRIPTION
**Pull request by ShyykoSerhiy**

### CODE REVIEW

File: lib/loader.js [hunk 1]
```diff
@@ -64,23 +64,28 @@ module.exports = function(content, map) {
...
-			if(idx > 0) { // idx === 0 is catched by isUrlRequest
-				// in cases like url('webfont.eot?#iefix')
-				urlRequest = url.substr(0, idx);
-				return "\" + require(" + loaderUtils.stringifyRequest(this, urlRequest) + ") + \"" +
-					url.substr(idx);
-			}
-			urlRequest = url;
-			return "\" + require(" + loaderUtils.stringifyRequest(this, urlRequest) + ") + \"";
-		}.bind(this));
+		if(!query.hasOwnProperty('import')) {
```
Comment (sokra) [inline lib/loader.js]: Please don't check for exisitance of query properties. Check for the value. `query.import !== false`. Document it as `css-loader?-url&-import`.

Elsewise this cases would be broken:

* `css-loader?url`
* `css-loader?{url: true}`

File: lib/processCss.js [hunk 2]
```diff
@@ -83,23 +86,27 @@ var parserPlugin = postcss.plugin("css-loader-parser", function(options) {
...
+								item.stringType = "";
+								delete item.innerSpacingBefore;
+								delete item.innerSpacingAfter;
+								var url = item.url;
+								item.url = "___CSS_LOADER_URL___" + urlItems.length + "___";
+								urlItems.push({
+									url: url
+								});
+							}
+							break;
```
Comment (sokra) [inline lib/processCss.js]: The break should be outside of the `if`, at the end of the case clause.

**Comment by sokra** [comment]
Please add test cases that validate the behavior.

**Comment by ShyykoSerhiy** [comment]
@sokra I've added some tests and fixed checks

**Comment by jasonslyvia** [comment]
Please merge this!

**Comment by xile611** [comment]
Please merge this!

**Comment by arcthur** [comment]
Please merge this!

**Comment by jasonHzq** [comment]
Please merge this!

**Comment by shelbeniskb** [comment]
please merge this!

**Comment by sokra** [comment]
> please merge this!

Instead of spamming `please merge this!` try to review the PR and comment with broken stuff. I. e. here were test regarding the `@import` handling missing. And no tests propably means: behavior is broken.

**Comment by vikalpgupta-birdeye** [comment]
@sokra This parameter works for url rewriting inside the stylesheet, but it also does not emit the files, can there be a parameter, in which we can specify such requirement

**Pull request merged**